src/components/dashboard/Rewards.js

```diff
@@ -8,17 +8,18 @@ import logger from '../../lib/logger/pino-logger'
 import SimpleStore from '../../lib/undux/SimpleStore'
 import { useDialog } from '../../lib/undux/utils/dialog'
 import { createIframe } from '../webView/iframe'
+import useOnPress from '../../lib/hooks/useOnPress'
 
 const log = logger.child({ from: 'RewardsTab' })
 
 const openInNewTab = false //isIOSWeb
-const RewardsTab = props => {
+const RewardsTab = ({ navigation }) => {
   const [token, setToken] = useState()
   const store = SimpleStore.useStore()
   const [showDialog] = useDialog()
 
   const getRewardsPath = () => {
-    const params = get(props, 'navigation.state.params', {})
+    const params = get(navigation, 'state.params', {})
```

Comment:
we have some refactorings for this page
once this PR will be merged, we have to discuss about those changes and try to make this component cross-platform: https://github.com/GoodDollar/GoodDAPP/pull/2214